features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/LifeCycle/Policies.jsx

```diff
@@ -1,167 +0,0 @@
-
-    changeTiers() {
-        this.setState({
-            loading: true,
-        });
-        const { api: { id: apiUUID } } = this.props;
-        const promisedApi = this.api.get(apiUUID);
-        promisedApi.then((response) => {
-            const apiData = JSON.parse(response.data);
-            apiData.policies = this.state.selectedPolicies;
-            const promisedUpdate = this.api.update(apiData);
-            promisedUpdate.then(() => {
-                this.setState({
-                    loading: false,
-                });
-                Alert.info('Lifecycle state updated successfully');
-            });
-        });
-    }
-
-    /**
-     *
-     *
-     * @returns
-     * @memberof Policies
-     */
-    render() {
-        const {
-            handleInputChange, api, policies, isAPIProduct, classes,
-        } = this.props;
-        const { loading } = this.state;
-        if (loading) {
-            return 'Loading . . .';
-        }
-        return (
-            <>
-                <FormControl className={classes.FormControl}>
-                    <InputLabel htmlFor='policy-selector'>
-                        <FormattedMessage
-                            id='Apis.Details.LifeCycle.Policies.business.plans'
-                            defaultMessage='Business Plans'
-                        />
-                    </InputLabel>
-                    <Select
-                        error={api.policies && api.policies.length === 0}
-                        fullWidth
-                        margin='none'
-                        multiple
-                        name='policies'
-                        value={api.policies || []}
-                        onChange={handleInputChange}
-                        input={<Input id='policy-selector' />}
-                        MenuProps={{
-                            PaperProps: {
-                                style: {
-                                    width: 200,
-                                },
-                            },
-                        }}
-                    >
-                        {policies.map((policy) => (
-                            <MenuItem
-                                key={policy.name}
-                                value={policy.name}
-                                style={{
-                                    fontWeight: policies.indexOf(policy.name) !== -1 ? '500' : '400',
-                                }}
-                            >
-                                {policy.name}
-                            </MenuItem>
-                        ))}
-                    </Select>
-                    <FormHelperText>
-                        {isAPIProduct
-                            ? (
-                                <FormattedMessage
-                                    id='Apis.Details.LifeCycle.Policies.select.plan.api.product'
-                                    defaultMessage='Select a plan for the API product.'
```

Comment:
@chanaka3d I think this file is in use ?

Reply:
This file is refered from source/src/app/components/Apis/Create/Default/APICreateDefault.test.jsx.

It seems we have to restore this file.